scripts/ci/github_actions/driver/setup_ci_dependencies.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -xe
+set +o pipefail
+
+python -m pip install --upgrade pip
+echo docker-hub==2.2.0 > dev-requirements.txt
+pip install -r dev-requirements.txt
+
+source /home/runner/.bash_profile
+cd common
+image_version=`yq eval .identity.version config.yaml`
```

Comment:
[image version](https://github.com/IBM/ibm-block-csi-driver/blob/bc36e1c6f3bf25ead92f54b89136a70bfcf716fc/Dockerfile-csi-controller#L18) is actually not necessarily the same as the driver image.
`image_version` -> `driver_version`

Reply:
I don't understand what you mean, do I need to get it from another place?

Reply:
I think he suggests renaming the variable

Reply:
IMO the variable name should be `driver_version`

Reply:
done